frontend/server/src/main/java/org/pytorch/serve/util/logging/QLogLayout.java

```diff
@@ -61,7 +72,7 @@
      * @return
      */
     @Override
-    public String format(LoggingEvent event) {
+    public String toSerializable(LogEvent event) {
         Object eventMessage = event.getMessage();
         String programName =
                 getStringOrDefault(System.getenv("MXNETMODELSERVER_PROGRAM"), "MXNetModelServer");
```

Comment:
Just noticed this in the code - separately, this could be fixed to use 'TS' in another PR/issue.